### ISSUE TITLE
Revise Darwin Core Namespace Policy to make it applicable across TDWG vocabulary standards

### ISSUE THREAD

**Issue by baskaufs** [tdwg/vocab#6]
The Term Change (Section 3) of the Darwin Core Namespace Policy [1] governs the way that changes are made to terms in the DwC vocabulary, and is modeled on the Dublin Core Namespace Policy [2].  Specific processes are described for handling errata, changing terms, and adding terms.  In practice, these processes are generally followed, although the TAG does not generally play the role specified in the policy.  The Namespace Policy does not specify a process for how changes to DwC other than term changes (e.g. changes to non-normative documentation that is part of the standard, e.g. the RDF Guide) - that will be raised as a separate issue.  For notes and disadvantages of the DwC Namespace Policy, see Sections 2.5.3 and 2.5.4 of the VoMag Report [3].  For the VoMag Report's recommendations, see Section 2.6.2 of the VoMag Report.  

The probable course of action will be to create a generic method for making changes to all TDWG vocabularies (Darwin Core, Audubon Core, and any future vocabulary standards) as part of the Vocabulary Maintenance Specification standard, then remove the specific DwC Term Change policy from the DwC Standard.

Resolving this issue is intended to be a major milestone for creation of the Vocabulary Maintenance Specification.  It is likely to be blocked by sub-issues involving details about the way the Namespace Policy works.  So please feel free to create separate sub-issues and I'll block this one on them.  

[1] http://rs.tdwg.org/dwc/terms/namespace/index.htm#classesofchanges
[2] http://dublincore.org/documents/2007/07/02/dcmi-namespace/
[3] https://github.com/tdwg/vocab/blob/master/gbif_TDWG_Vocabulary_Management_Task_Group_en_v1.0.pdf

**Comment by baskaufs** [comment]
Blocked on Issue #8

**Comment by baskaufs** [comment]
Block on Issue #12

**Comment by baskaufs** [comment]
Blocked on Issue #13

**Comment by baskaufs** [comment]
No longer blocked by closed issues #8, #12, and #13

**Comment by baskaufs** [comment]
This issue is now resolved.  The various blocking sub-issues, such as the role of the TAG and making changes to parts of the standard other than terms have been addressed in the draft Vocabulary Maintenance Specification https://github.com/tdwg/vocab/blob/master/maintenance-specification.md .  What's left of the Darwin Core Namespace document after pulling out the term change policy is rather sparse. The wreckage can be found at the bottom of the draft Vocabulary Management Spec.  It will have to be cleaned up and edited if the Vocab Management Spec is adopted.  

The contributors to the Darwin Core Namespace Policy document have been added to the list of contributors to the draft spec in order to acknowledge their contributions.  I hacked a fair bit of text directly from the DwC Namespace doc and also borrowed ideas freely from the Darwin Core cover page (http://rs.tdwg.org/dwc/), particularly in writing section 3.1 where I laid out the overarching considerations for making changes to vocabularies.  Thanks DwC TG for laying the foundation for this!
